tests/python/test_multidevice.py

```diff
@@ -43,7 +43,9 @@ def test_pointwise(multidevice_test):
     # scalar inputs isn't supported;
     class MultiDeviceModel(FusionDefinition):
         def definition(self):
-            self.t0 = self.define_tensor((2, 4), (False, False), dtype=DataType.Float)
+            self.t0 = self.define_tensor(
+                (num_devices, 4), (False, False), dtype=DataType.Float
```

Comment:
Line 54 needs a fix too...
```
mesh = self.sched._create_device_mesh((0, 1))
```

Reply:
GitHub CI should capture that. It runs on 4 GPUs.

Reply:
sorry I was just being sarcastic and that's not effective communication.

I notice we also have `mpirun -np 1`. which this would still fail. I need one more line of change...

Reply:
It took quite a while to rebuild on those slower machine with 8 gpus.... I was just complaining about that.